.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+jobs:
+  pypi:
+    name: Publish sdist and wheel to PyPI
+    runs-on: ubuntu-20.04
+    environment: Release
```

Comment:
N.B.: This both exposes the PYPI_{USERNAME ,PASSWORD} secrets, which are configured here: https://github.com/pantsbuild/pex/settings/environments, and requires a committer approve the release since I configured the `Release` environment to have required reviewers: https://docs.github.com/en/actions/reference/environments#required-reviewers.

We'll see what that's like and if it should stay.